utils/resolve.js

```diff
@@ -174,14 +172,26 @@ function resolverReducer(resolvers, map) {
   throw err
 }
 
-function getBaseDir(sourceFile) {
-  return pkgDir.sync(sourceFile) || process.cwd()
-}
 function requireResolver(name, sourceFile) {
   // Try to resolve package with conventional name
-  let resolver = tryRequire(`eslint-import-resolver-${name}`, sourceFile) ||
+  let resolver =
+    /**
+     * normal abbreviation for third-party package
+     * like config `webpack` convert to `eslint-import-resolver-webpack`
+     */
+    tryRequire(`eslint-import-resolver-${name}`, sourceFile) ||
+    /**
+     * normal full-name for third-party package or absolute path
+     * like config `@xxx/webpack-resolver` or `/your/absolute/path/resolver.js`
+     */
     tryRequire(name, sourceFile) ||
-    tryRequire(path.resolve(getBaseDir(sourceFile), name))
```

Comment:
looks like this package has been used since 693be1788, the creation of this package.

I'm a bit concerned about its removal even though tests seem to be passing. @benmosher, any idea why this package is used here specifically?